ccd-definition/FixedLists.json

```diff
@@ -597,5 +597,26 @@
     "ListElementCode": "NONE",
     "ListElement": "None",
     "DisplayOrder": 7
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "StatementOfServiceSentBy",
+    "ListElementCode": "EMAIL",
+    "ListElement": "email",
```

Comment:
Do we not need title case on each of the three listElement?

Reply:
Sam asked for that before he wanted all list elements to be declared as enums.

Reply:
I mean on the listElements, should email not be Email etc